src/frontend/src/App.tsx

```diff
@@ -1,7 +1,13 @@
 import { AppRoutes } from './routes'
+import { ProvideAuth } from './features/auth/cognito-auth';
+
+function App() {
```

Comment:
以下のような形になるのが理想
App.tsxにおいて個別にProviderは適用しない
```suggestion
function App() {
  return (
    <AppProvider>
      <AppRoutes />
    </AppProvider>
  );
}
```